kse/src/org/kse/crypto/privatekey/PrivateUtils.java

```diff
@@ -0,0 +1,222 @@
+package org.kse.crypto.privatekey;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.security.PrivateKey;
+import java.security.interfaces.DSAPrivateKey;
+import java.security.interfaces.RSAPrivateCrtKey;
+import java.text.MessageFormat;
+import java.util.ResourceBundle;
+
+import javax.swing.JFrame;
+import javax.swing.JOptionPane;
+
+import org.kse.crypto.CryptoException;
+import org.kse.crypto.Password;
+import org.kse.gui.dialogs.importexport.DExportPrivateKeyOpenSsl;
+import org.kse.gui.dialogs.importexport.DExportPrivateKeyPkcs8;
+import org.kse.gui.dialogs.importexport.DExportPrivateKeyPvk;
+import org.kse.gui.preferences.ApplicationSettings;
+
+public class PrivateUtils {
```

Comment:
Please call this class `PrivateKeyUtils` and in general try to adhere to the Java convention of adding a JavaDoc comment to everything that is public.